build/templates/README.md

```diff
@@ -293,145 +304,148 @@ Note that there are sometimes backward-incompatible changes to SQL features betw
+| `tls.selfSigner.image.tag`                                | Image tag to use for self signing TLS certificates                                                                                                                                                                                                                                                                                       | `0.1`                                                  |
+| `tls.selfSigner.image.pullPolicy`                         | Self signing TLS certificates container pull policy                                                                                                                                                                                                                                                                                      | `IfNotPresent`                                         |
+| `tls.selfSigner.image.credentials`                        | `registry`, `user` and `pass` credentials to pull private image                                                                                                                                                                                                                                                                          | `{}`                                                   |
+| `networkPolicy.enabled`                                   | Enable NetworkPolicy for CockroachDB's Pods                                                                                                                                                                                                                                                                                              | `no`                                                   |
+| `networkPolicy.ingress.grpc`                              | Whitelist resources to access gRPC port of CockroachDB's Pods                                                                                                                                                                                                                                                                            | `[]`                                                   |
+| `networkPolicy.ingress.http`                              | Whitelist resources to access gRPC port of CockroachDB's Pods                                                                                                                                                                                                                                                                            | `[]`                                                   |
```

Comment:
Nit: This is one line change for `conf.wal-failover` so if you carefully change then the diff should only show one line as follows:

This would be helpful if we look at this PR later.

Reply:
I actually added `conf.log` and `conf.store.count` too.

Also, `conf.locality` was incorrectly placed, so had to move it correctly. I had carefully made these changes and the resulting diff is what you see.

I realize that we are not strictly following the lexicographical order (e.g. `conf.port`, `conf.http-port`), so I'll undo this change and see what diff it generates.